.lgtm.yml

```diff
@@ -0,0 +1,12 @@
+---
+# This file contains configuration for the LGTM.com static code analysis tool.
+# The python-blosc GitHub repository is continuously scanned by the LGTM.com
+# tool for security and/or code vulnerabilities. You can find the alerts here:
+# https://lgtm.com/projects/g/Blosc/python-blosc/
+
+path_classifiers:
+  generated:
+    # suppress alerts in vendored files
+    - cpuinfo.py
+    - blosc/_version.py
+    - blosc/c-blosc
```

Comment:
That means that c-blosc alerts should be suppressed, but I am seeing several alerts in https://lgtm.com/projects/g/Blosc/python-blosc/.  Am I missing something?

Reply:
LGTM.com scans the default branch of repositories
- I expect alerts related to vendored code to disappear when this file is merged in the default branch `main`.
- Other alerts may be relevant. You can hose to fix them or ignore later on.